Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -949,75 +969,67 @@ gdjs.TweenRuntimeBehavior.makeNewTweenable = function(runtimeScene) {
   runtimeScene.shiftyJsScene.add(tweenable);
   return tweenable;
 };
-
 /**
  * Static function to remove a Tweenable from a scene.
  *
  * @param {gdjs.RuntimeScene} runtimeScene
  * @param {shifty.Tweenable} tweenable
  */
+
 gdjs.TweenRuntimeBehavior.removeFromScene = function(runtimeScene, tweenable) {
   if (!runtimeScene.shiftyJsScene) return;
-
   runtimeScene.shiftyJsScene.remove(tweenable);
-};
-
-// Callbacks called to pause/resume Shifty scene when a gdjs.RuntimeScene
+}; // Callbacks called to pause/resume Shifty scene when a gdjs.RuntimeScene
 // is paused/resumed
 
 /**
  * Stop and "destroy" all the tweens when a scene is unloaded.
  * @private
  */
+
 gdjs.TweenRuntimeBehavior.gdjsCallbackRuntimeSceneUnloaded = function(
   runtimeScene
 ) {
-  if (!runtimeScene.shiftyJsScene) return;
-
-  // Stop and explictly remove all tweenables to be sure to drop
+  if (!runtimeScene.shiftyJsScene) return; // Stop and explictly remove all tweenables to be sure to drop
```

Comment:
This change is quite bad because it moved the comment at a place that is not related to the comment, making it misleading